chain/beacon/store.go

```diff
@@ -18,11 +17,14 @@ import (
 	"github.com/drand/drand/metrics"
 )
 
+// CallbackFunc defines the callback type that's accepted by CallbackStore
+type CallbackFunc func(b *chain.Beacon, closing bool)
```

Comment:
I'd suggest calling this `closed` to ensure no confusion that something ought to happen when finding it `closing`, but I'll not block merging cos I'd rather get it in testnet asap to try it out